06_discord_bot/06_discord_bot.md

```diff
@@ -0,0 +1,242 @@
+verrez les informations de votre Bot, dont son pseudo (`username`) et son avatar que vous pourrez modifier selon vos envies, et surtout son token qui **doit toujours rester secret !** (sauf pour vous). Si votre token fuite (sur
+internet par exemple), changez le ! Gardez le dans un coin, on va aussi en
+avoir besoin pour plus tard. Et ne vous inquiétez pas pour le token de mon bot sur l'image, je l'ai changé après avoir mis ça ici.
+
+[comment]: ![image](ims/im9.png)
+<center>
+<img src="ims/im9.png" alt="centered image" width="500"/>
+</center>
+
+Voilà, votre bot est créé. Il faut maintenant le connecter à discord et lui donner du code à lire et exécuter.
+
+
+Connecter son Bot
+==============
+
+Pour interagir avec votre bot, lui donner des instructions et le tester, vous allez avoir besoin d’y accéder via discord. Nous allons l’inviter dans un
+serveur discord créé pour l'occasion. Vous pourrez supprimer ce serveur à la fin des tests si cela vous chante. Ouvrez discord (l'application que vous avez l'habitude d'utiliser) et créez un serveur de test en appuyant sur le `+` dans la barre de gauche en bas de vos serveurs.
+
+Ensuite, dans votre navigateur de recherche (par exemple firefox ou google chrome), tapez le lien suivant en remplaçant \[votre ID\] par l’ID de votre bot, sans guillemet. Normalement, vous avez noté cet ID dans un coin lors de la partie "Création du Bot".
+
+    https://discordapp.com/oauth2/authorize?client_id=[votre ID]&scope=bot&permissions=0
+
+Vous devriez voir apparaître une page comme celle ci.
+
+[comment]: ![image](ims/im3.png)
+<center>
+<img src="ims/im3.png" alt="centered image" width="350"/>
+</center>
+
+Sélectionnez le serveur dans lequel vous voulez inviter votre Bot, c'est à dire le serveur Test que nous venons de créer, et cliquez sur le bouton
+“Authorize”. Dans quelques instants, vous devriez voir votre Bot
+apparaître en tant que membre de ce serveur. Pour l’instant,
+il est hors-ligne. C’est tout à fait normal, car nous ne lui avons
+pas encore donné de code à exécuter et nous ne l’avons pas lancé. C’est
+ce que nous allons faire maintenant.
+
+Coder son Bot
+=============
+
+Il est temps d’écrire le fichier python qui contiendra votre code.
+Créez-en un où vous voulez à l’aide de votre éditeur de texte préféré.
+Par défaut, si vous n’en connaissez aucun, vous pouvez utiliser gedit
+sous linux. Nommez-le comme vous voulez avec l’extension `.py`, par
+exemple `bot.py`, et remplissez le de la manière suivante :
+
+``` python
+import discord
+import asyncio
+from discord.ext import commands
+bot = commands.Bot(command_prefix='!', description="ce que vous voulez en description")
+#vous pouvez changer la commande '!' en autre chose si vous le souhaitez
+
+@bot.event
+async def on_ready():
+    print("Logged in as")
+    print(bot.user.name)
+
+#ici figureront les commandes et instructions pour votre bot
+
+bot.run("token") #vous devez ici rentrer le token de votre bot, que vous avez trouvé plus haut.
+```
```

Comment:
Lines longer than ~80 columns don't fit in the page width of the generated PDF:
[06_discord_bot.pdf](https://github.com/prologin/gcc-resources/files/5118765/06_discord_bot.pdf)